sdk/Sdk/ExtensionsCsprojGenerator.cs

```diff
@@ -49,14 +49,21 @@ private void WriteExtensionsCsProj(string filePath)
             File.WriteAllText(filePath, csprojContent);
         }
 
+
```

Comment:
nit: were the extra lines here by accident?

Reply:
yeah, let me fix that.